oxidation/libparsec/crates/client_types/src/local_device_file.rs

```diff
@@ -101,6 +101,24 @@ impl DeviceFile {
     }
 }
 
+#[serde_as]
+#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
```

Comment:
This struct is only use to read `LegacyDevice` file that then are converted to `DeviceFile`.
We don't need to serialize the struct

Reply:
Since you're using `Serialize` in test for the roundtrip use
```suggestion
#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
#[cfg_attr(test, derive(Serialize))]
```

Reply:
I did it but we can't do these tests as `integration` tests because `cfg(test)` is only enabled when tests in the current crate are being compiled.

Reply:
Done.